### PR TITLE
controlpanel cleanup

### PR DESCRIPTION
**Pull request by jensens**
another code cleanup

### CODE REVIEW

**Comment by tisto** [comment]
@jensens Jenkins is red on the pr. Please re-run and make sure tests pass...

File: CHANGES.rst [hunk 1]
```diff
@@ -183,6 +183,9 @@ Bug fixes:
   [gforcada, ivanteoh, maurits]
 
 
+- Cleanup all controlpanel related code (pep7, utf8headers, style, readability)
```
Comment (hvelarde) [inline CHANGES.rst]: pep8

File: setup.py [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
-        "License :: OSI Approved :: GNU General Public License (GPL)",
-        "Operating System :: OS Independent",
-        "Programming Language :: Python",
-        "Programming Language :: Python :: 2.7",
-        "Topic :: Internet :: WWW/HTTP :: Dynamic Content",
+        'Environment :: Web Environment',
+        'Framework :: Plone',
+        'Framework :: Plone :: 5.1',
+        'Framework :: Zope2',
+        'License :: OSI Approved :: GNU General Public License (GPL)',
```
Comment (hvelarde) [inline setup.py]: license information is wrong, BTW: it should be GPLv2.

**Comment by jensens** [comment]
@tisto, yes, I know, back to WIP

**Comment by jensens** [comment]
need to start this again, but not now.

**Pull request closed without merging**